utils/roadie-backstage-entity-validator/package.json

```diff
@@ -45,7 +45,6 @@
   },
   "devDependencies": {
     "@backstage/cli": "^0.25.0",
-    "@vercel/ncc": "^0.27.0",
```

Comment:
this is used in the `prepare` step above

Reply:
Yea, just realized, needed a version bump to comply the new OpenSSL defaults